### ISSUE TITLE
Menu doesn't close when clicking start multiple times.

### ISSUE THREAD

**Issue by vXtreniusX** [Tuxemon/Tuxemon#832]
Instead, it opens another menu, overlapping the previous one.

**Comment by bitcraft** [comment]
Is this the main menu, right after the logo?

**Comment by vXtreniusX** [comment]
No, the in game menu, activated by esc key or start (on controllers)

**Comment by bitcraft** [comment]
Ok tbanks

**Comment by kerizane** [comment]
I can't reproduce this.
I'm running on desktop with python3.7 against commit: 49d3aef101b277603f044ffeb7934ea3d2ab9c10

**Comment by vXtreniusX** [comment]
> I can't reproduce this.
> I'm running on desktop with python3.7 against commit: [49d3aef](https://github.com/Tuxemon/Tuxemon/commit/49d3aef101b277603f044ffeb7934ea3d2ab9c10)

I used xbox 360 controller. (OS: Manjaro 20 if I remember correctly)

**Comment by kerizane** [comment]
Does it happen with the keyboard?

**Comment by vXtreniusX** [comment]
> 
> 
> Does it happen with the keyboard?

No, only with the controller.

**Comment by ultidonki** [comment]
On a controller, you have 2 buttons, buttons.START and buttons.BACK. The BACK button opens and closes it OK. The START button keeps opening it over and over again.
On the keyboard, you can only open the menu using ESCAPE (maps to buttons.BACK), which opens and closes it OK.

It happens because:
In tuxemon/menu/menu.py, the menu is only closed if buttons.BACK is pressed, if you press buttons.START it doesn't close the menu, or handle the event, so the event is passed to the worldstate to be handled. 
In tuxemon/states/world/worldstate.py, both BACK and START are mapped to open the menu, INTENTIONS.WORLD_MENU.
So when the START event is passed to menu.py, nothing happens so it gets passed to the worldstate, which re-opens the world menu again. 

You can reproduce it by mapping a key on the keyboard to buttons.START:
Eg. in your tuxemon.cfg add under '[Controls]':
`start = e`
Now, when you press 'e', you can reproduce the problem.

I'm not sure if we should either:
- Allow closing a menu if you press buttons.START
- Capture the buttons.START event in menu.py, and do nothing with it but don't pass it to the next active state. 
- Remove buttons.START and map it so both buttons only use buttons.BACK
